src/App.tsx

```diff
@@ -1,14 +1,41 @@
 /* eslint-disable max-len */
-import React from 'react';
+import { useEffect, useState } from 'react';
 import 'bulma/css/bulma.css';
 import '@fortawesome/fontawesome-free/css/all.css';
 
 import { TodoList } from './components/TodoList';
 import { TodoFilter } from './components/TodoFilter';
 import { TodoModal } from './components/TodoModal';
 import { Loader } from './components/Loader';
+import { Todo } from './types/Todo';
+import { getTodos } from './api';
 
 export const App: React.FC = () => {
+  const [todos, setTodos] = useState<Todo[]>([]);
+  const [filterBy, setFilterBy] = useState<string>('All');
```

Comment:
No need to write primitive types. The same in lines 16-17.
```suggestion
  const [filterBy, setFilterBy] = useState('All');
```
But in this case better create enum for filter type and also use it in `swith case`.